pkg/skaffold/debug/transform.go

```diff
@@ -200,3 +213,24 @@ func encodeConfigurations(configurations map[string]map[string]interface{}) stri
 	}
 	return string(bytes)
 }
+
+func describe(obj runtime.Object) (group, version, kind, description string) {
+	// get metadata/name; shamelessly stolen from from k8s.io/cli-runtime/pkg/printers/name.go
+	name := "<unknown>"
+	if acc, err := meta.Accessor(obj); err == nil {
+		if n := acc.GetName(); len(n) > 0 {
+			name = n
+		}
+	}
+
+	gvk := obj.GetObjectKind().GroupVersionKind()
+	group = gvk.Group
+	version = gvk.Version
+	kind = gvk.Kind
+	if group == "" {
+		description = fmt.Sprintf("%s/%s", strings.ToLower(kind), name)
+	} else {
+		description = fmt.Sprintf("%s.%s/%s", strings.ToLower(kind), group, name)
+	}
+	return
```

Comment:
we dont return anything here or am i missing some hidden go mojo that happens here?

Reply:
[_named return values_](https://tour.golang.org/basics/7)